### PR TITLE
Update ServiceAccountIssuerDiscovery docs for GA

### PR DESCRIPTION
**Pull request by mtaufen**
Updates the SAID feature docs now that the feature is GA in 1.21.

See: https://github.com/kubernetes/enhancements/issues/1393

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md [hunk 1]
```diff
@@ -323,11 +323,10 @@ The application is responsible for reloading the token when it rotates. Periodic
 
 ## Service Account Issuer Discovery
 
-{{< feature-state for_k8s_version="v1.20" state="beta" >}}
+{{< feature-state for_k8s_version="v1.20" state="stable" >}}
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/configure-service-account.md:326]: I think this should be:
```suggestion
{{< feature-state for_k8s_version="v1.21" state="stable" >}}
```

**Review by sftim — DISMISSED** [review]
@mtaufen thanks

I think it needs one correction - see inline feedback.

**Comment by liggitt** [comment]
lgtm after the for_k8s_version comment is addressed

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md (hunk 1 above)
Comment (mtaufen) [reply content/en/docs/tasks/configure-pod-container/configure-service-account.md]: Done.

**Comment by mtaufen** [comment]
Updated.

**Review by irvifa — COMMENTED** [review]
/approve

**Pull request merged**